install/lib/class.FroxlorInstall.php

```diff
@@ -234,14 +234,14 @@ private function _doInstall()
 				$fatal_fail = true;
 			}
 		}
-		$version_server = $db_root->getAttribute(PDO::ATTR_SERVER_VERSION);
-		$sql_mode = 'NO_ENGINE_SUBSTITUTION';
-		if (version_compare($version_server, '8.0.11', '<')) {
-			$sql_mode .= ',NO_AUTO_CREATE_USER';
-		}
-		$db_root->exec('SET sql_mode = "'.$sql_mode.'"');
 		
 		if (! $fatal_fail) {
+			$version_server = $db_root->getAttribute(PDO::ATTR_SERVER_VERSION);
```

Comment:
the move is correct yes...settings the mode only makes sense if connection works